go.mod

```diff
@@ -25,12 +25,12 @@ require (
 	github.com/rancher/wrangler/v2 v2.1.2
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.7.0
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.9.0
 	// Do not bump testcontainers version. This will cause conflicts with go.opentelemetry.io because of a very old version
 	// found in steve. This is used just for testing.
```

Comment:
```suggestion
```
`rancher/steve` is no longer a dependency since [this commit](https://github.com/rancher/gitjob/commit/cc95fddcdd98d5f910977776f6da78c8c5c87a21).